scripts/generate_unknown_objects.py

```diff
@@ -0,0 +1,105 @@
+import argparse
+import logging
+from os.path import isdir
+from pathlib import Path
+from shutil import rmtree, copytree
+from typing import Mapping, Sequence
+
+import yaml
+
+situation_num: int = 0
+
+
+def get_situation_num(situation_string: Path) -> int:
+    """Given a Path for a situation directory, return the associated number"""
+    return int(str(situation_string).split('_')[-1])
+
+
+def copy_directory(start_path: Path, end_path: Path) -> None:
+    """Function to recursively copy directory from start_path to end_path"""
+    copytree(str(start_path), str(end_path))
+
+
+def copy_all_directories(output_dir: Path, dirs_to_move: Sequence[Path]):
+    """Move all directories in dirs_to_move to their corresponding location in the target directory"""
+    global situation_num
+    sorted_dirs = sorted(dirs_to_move, key=get_situation_num)
+    for directory in sorted_dirs:
+        copy_directory(directory, output_dir / f'situation_{situation_num}')
+        situation_num += 1
+
+
+def main() -> None:
+    parser = argparse.ArgumentParser(
+        description="Utility to filter out specified objects from curriculum"
+    )
+    parser.add_argument('--input-dir', type=Path, help="Directory with curriculum training samples")
+    parser.add_argument('--output-dir', type=Path, help="Directory to write remaining samples to")
+    parser.add_argument("--curriculum-name", required=False, type=str, help="Name of training curriculum")
```

Comment:
I don't think it's necessary to specify "training" here. It doesn't have to be. It's just that that's all we're using it for right now.

Reply:
I think it is confusing to specify `training` curriculum here, because there's no reason that the output must be used for training. I'd rather call it `Name of output curriculum`.

Reply:
Ok, I can update that.